plugins/generic/usageEvent/UsageEventPlugin.inc.php

```diff
@@ -286,6 +286,18 @@ function _buildUsageEvent($hookName, $args) {
 			$request, null, $canonicalUrlPage, $canonicalUrlOp, $canonicalUrlParams
 		);
 
+		// Make sure we log the server name and not aliases.
+		$configBaseUrl = Config::getVar('general', 'base_url');
+		$requestBaseUrl = $request->getBaseUrl();
+		if ($requestBaseUrl !== $configBaseUrl) {
+			// Make sure it's not an url override (no alias on that case).
+			if (!in_array($requestBaseUrl, Config::getContextBaseUrls()) && 
+					$requestBaseUrl !== Config::getVar('general', 'base_url[index]')) {
```

Comment:
no, it's always searching for the entire base url string... I just want to know if the current request base url contains a server alias or not. So, if that's not equal to any base_url config (including the override ones) defined in config.inc.php, then I consider as an alias. Does that makes sense?